app/models/mentor/discussion.rb

```diff
@@ -55,6 +57,8 @@ def self.between(mentor:, student:)
 
   after_save_commit do
     solution.update_mentoring_status! if previous_changes.key?('status')
+    update_num_solutions_mentored! if previous_changes.key?('status')
+    update_satisfaction_rating! if previous_changes.key?('rating')
```

Comment:
So a couple of things:
1. These are going to need to go in a job (you can put both in one job called `UpdateMentorStatsJob` or something). I've checked on Bastion, and they'd add 60ms to the request, which I think is too long to have inline.
2. I don't think all the logic in `update_num_solutions_mentored!` etc belongs in discussion. I think each method should be it's own command. I think they can both be at the apex of `app/commands/mentor` (e.g. `Mentor::UpdateSatisfactionRating`)